packages/css/src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-tabs__list {
+  border-bottom-color: var(--amsterdam-tabs-border-color);
+  border-bottom-style: solid;
+  border-bottom-width: var(--amsterdam-tabs-border-width);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  position: relative;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    overflow: visible;
+  }
+
+  &::-webkit-scrollbar {
+    background: transparent;
+    height: 0;
+    scrollbar-width: none;
+  }
+}
+
+.amsterdam-tabs__button {
+  background-color: var(--amsterdam-tabs-button-background-color);
+  border: var(--amsterdam-tabs-button-border);
+  color: var(--amsterdam-tabs-button-color);
+  cursor: var(--amsterdam-action-activate-cursor);
+  display: inline-block;
```

Comment:
This is the initial value; however, as a flex item, its `display` value will be set to `block` anyway.
```suggestion
```